### DISCUSSION TITLE
Custom field => entries dissappear

### DISCUSSION THREAD

**Discussion by kevin-vanb** [liedekef/events-made-easy#105] [General]
Hello Franky,

in the test environment where I implemented the code (https://github.com/liedekef/events-made-easy/discussions/89) for the search on empty custom fields, I see the following behavior

1) Add new member, all well in listing members
2) Add new custom field (member purpose), required and searchable dropdown
3) Use in the the member registration form, still empty (on purpose) in the next step I want to test on empty values
4) Member is not listed anymore

Any idea if this change (https://github.com/liedekef/events-made-easy/discussions/89)  created this behavior?

Kind regards,
Kevin

**Comment by liedekef** [comment]
"Add new custom field (member purpose), required and searchable dropdown"
==> how can a required field be empty???

**Reply by kevin-vanb** [reply]
Because it's information we want to know about members, but since it was not initially intended, we added the field afterwards. However, new members need to fill it in the moment they register themselves :-)

**Reply by liedekef** [reply]
ok, since you added the field afterwards, the value is non-existing for existing members (not empty, but not-existing at all). That's why the field search will not show anything.

**Reply by kevin-vanb** [reply]
But its not a specific search I am performing, unless the listing is also a search? However there is a valid member in the system right now.
![afbeelding](https://user-images.githubusercontent.com/123742526/225117681-847f245b-b3a1-4776-ab66-c18b8dc0b801.png)

**Reply by kevin-vanb** [reply]
Ok, I found it. It was not required anymore, but since it was set as searchable, that caused not being visible in the list. Changing that solved the issue. Apologies :-) 
So I guess looking for empty values is difficult since when the field is searchable, no members with empty values are displayed ;-)
